MIGRATION.md

```diff
@@ -743,36 +747,102 @@ Instead, use `export const parameters = {};` and `export const decorators = [];`
+
+If you were using `.stories.mdx` files to write stories, we encourage you to move the stories to a CSF file, and *attach* an `.mdx` file to that CSF file to document them. You can use the `Meta` block to attach a MDX file to a CSF file, and the `Story` block to render the stories:
 
 ```mdx
-import { Meta } from '@storybook/blocks';
+import { Meta, Story } from '@storybook/blocks';
 import * as ComponentStories from './some-component.stories';
 
 <Meta of={ComponentStories} />
+
+<Story of={ComponentStories.Primary} />
 ```
 
-You can create as many docs entries as you like for a given component. Note that if you attach a docs entry to a component it will replace the automatically generated entry from `DocsPage` (See below).
+You can create as many docs entries as you like for a given component. Note that if you attach a docs entry to a component it will replace the automatically generated entry from Autodocs.
 
 By default docs entries are listed first for the component. You can sort them using story sorting.
 
-#### Referencing stories in docs files
+
+#### Unattached docs files
+
+In Storybook 6.x, to create a unattached docs MDX file (that is, one not attached to story or a CSF file), you'd have to create a `.stories.mdx` file, and describe its location with the `Meta` doc block:
+
+```mdx
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Introduction" />
+```
+
+In 7.0, things are a little simpler -- you should call the file `.mdx` (drop the `.stories`). This will mean behind the scenes there is no story attached to this entry. You may also drop the `title` and use autotitle (and leave the `Meta` component out entirely, potentially).
+
+
+#### Doc Blocks
+
+Additionally to changing the docs information architecture, we've updated the API of the doc blocks themselves to be more consistent and future proof.
+
+**General changes**
+
+- Each block now uses `of={}` as a primary API -- where the argument to the `of` prop is a CSF or story *export*.
+
+- When you've attached to a CSF file (with the `Meta` block, or in Autodocs), you can drop the `of` and the block will reference the first story or the CSF file as a whole.
+
+- Most other props controlling rendering of blocks now correspond precisely to the parameters for that block [defined for autodocs above](#autodocs-changes).
+
+##### Meta block
+
+The primary change of the `Meta` block is the ability to attach to CSF files with `<Meta of={}>` as described above.
+
+##### Description block, `parameters.notes` and `parameters.info`
+
+In 6.5 the Description doc block accepted a range of different props, `markdown`, `type` and `children` as a way to customize the content.
+The props have been simplified and the block now only accepts an `of` prop, which can be a reference to either a CSF file, a default export (meta) or a story export, depending on which description you want to be shown. See TDB DOCS LINK for a deeper explanation of the new prop.
```

Comment:
what does it mean to reference "a CSF file" in this context? `import * from Foo.stories.js`?

Reply:
yes, or to be more precise `import * as FooStories from Foo.stories.js`